src/dpg_slam/dpg_slam.h

```diff
@@ -44,14 +45,38 @@ namespace dpg_slam {
      */
     class occupancyGrid {
     public:
+    
+    // Constructor without any nodes 
+    occupancyGrid(const DpgParameters &dpg_parameters, const PoseGraphParameters &pose_graph_parameters) :
+	      dpg_parameters_(dpg_parameters), pose_graph_parameters_(pose_graph_parameters) {}	    
+ 
+    // Constructor for combining two occupancy grids.
+    occupancyGrid(const occupancyGrid &grid1, const occupancyGrid &grid2, const DpgParameters &dpg_parameters, 
+	      const PoseGraphParameters &pose_graph_parameters) : dpg_parameters_(dpg_parameters), 
+    	      pose_graph_parameters_(pose_graph_parameters), min_cell_x_(INT_MAX), max_cell_x_(INT_MIN),
+               min_cell_y_(INT_MAX), max_cell_y_(INT_MIN), include_inactive_(false), include_added_(true), 
+	       include_static_(true) {
```

Comment:
I think you left off is_single_node_grid() here

Reply:
I think we can remove this variable overall. I kept around to see if I'd use it later, but it's unused anyway.